packages/define-options/src/index.ts

```diff
@@ -20,7 +20,7 @@ function resolveOption(
   const version = options.version || detectVueVersion()
   return {
     include: [REGEX_VUE_SFC, REGEX_SETUP_SFC].concat(
-      version === 2 && framework === 'webpack' ? REGEX_VUE_SUB : []
```

Comment:
I think we can remove them from all the packages

Reply:
If so, we need to change it like:

```ts
include: [REGEX_VUE_SFC, REGEX_SETUP_SFC, REGEX_VUE_SUB]
```